resources/views/backend/dashboard2.blade.php

```diff
@@ -0,0 +1,532 @@
+@extends('layouts.dash')
+@push('css')
+<link rel="stylesheet" href="{{ asset('css/dashboard.css') }}">
+<title>ExpenseNg - Dashboard</title>
+@endpush
+
+@section('content')
+<div class="dashboard-wrapper">
+    <div class="dashboard-ecommerce">
+        <div class="container-fluid dashboard-content ">
+         <!-- ============================================================== -->
+            <!-- Flash Messages  -->
```

Comment:
Why do we have 2 dashhboard?

Reply:
It was the previous dashboard. I will delete it now